runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemWithCharge.java

```diff
@@ -225,7 +217,11 @@
 	WSKIN1(WATERSKIN, WATERSKIN1, 1),
 	WSKIN2(WATERSKIN, WATERSKIN2, 2),
 	WSKIN3(WATERSKIN, WATERSKIN3, 3),
-	WSKIN4(WATERSKIN, WATERSKIN4, 4);
+	WSKIN4(WATERSKIN, WATERSKIN4, 4),
+	GREST1(GUTHIX_REST,GUTHIX_REST1, 1),
```

Comment:
Checkstyle is complaining about this. Make sure there's a space after each comma in the added lines.

```suggestion
	GREST1(GUTHIX_REST, GUTHIX_REST1, 1),
```